doc/guides/device_mgmt/ota.rst

```diff
@@ -0,0 +1,84 @@
+.. _ota:
+
+Over-the-Air Update
+###################
+
+Overview
+********
+
+Over-the-Air (OTA) Update is a method for delivering firmware updates to remote
+devices using a network connection. Although the name implies a wireless
+connection, updates received over a wired connection (such as Ethernet)
+are still commonly referred to as OTA updates. This approach requires server
+infrastructure to host the firmware binary and implement a method of signaling
+when an update is available. Security is a concern with OTA updates; firmware
+binaries should be cryptographically signed and verified before flashing.
+
+The :ref:`dfu` section discusses upgrading Zephyr firmware using MCUboot. The
+same method can be used as part of OTA. The binary is first downloaded
+into an unoccupied code partition, then flashed using the `MCUboot bootloader`_
+process.
```

Comment:
I find this terminology a bit weird for MCUboot users, it would be better to say something like ", then upgraded to using the ..."

Reply:
Also instead of pointing to the MCUboot website, you could just refer to the Zephyr docs which already have some MCUboot specific information with: ref:\`mcuboot`

Reply:
> ", then upgraded to using the ..."

Got it, thanks!

> Also instead of pointing to the MCUboot website, you could just refer to the Zephyr docs which already have some MCUboot specific information with: ref:`mcuboot`

I looked and I don't see a Zephyr docs page on MCUboot. Am I just missing it? The DFU page which I've already linked in this paragraph discuses it.

Reply:
The MCUboot reference I mentioned is actually a section inside the DFU page, you can link to it using ":ref:\`mcuboot`"

Reply:
Thanks for the clarification, I've pushed this change.